src/framework/ui/types/mnemonicstring.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2022 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+#ifndef MU_UI_MNEMONICSTRING_H
+#define MU_UI_MNEMONICSTRING_H
+
+#include <QString>
+
+#include "types/translatablestring.h"
+
+namespace mu::ui {
+class MnemonicString
```

Comment:
I think better move this type to global module, near TranslatableString

Reply:
Yeah, that's probably the best way to handle the linking problems. I'll do that.